tool/planet/main.go

```diff
@@ -162,6 +162,7 @@ func run() error {
 		cagentEtcdKeyFile            = cagent.Flag("etcd-keyfile", "TLS key file used to secure etcd communication").String()
 		cagentElectionEnabled        = Bool(cagent.Flag("election-enabled", "Boolean flag to control if the agent initially starts with election participation on").OverrideDefaultFromEnvar(EnvElectionEnabled))
 		cagentDNSUpstreamNameservers = List(cagent.Flag("nameservers", "List of additional upstream nameservers to add to DNS configuration as a comma-separated list of IPs").OverrideDefaultFromEnvar(EnvDNSUpstreamNameservers))
+		cagentDNSLocalNameservers    = List(cagent.Flag("local-nameservers", "List of addresses the local nameserver is listening on").OverrideDefaultFromEnvar(EnvDNSLocalNameservers).Default(DefaultDNSAddress))
```

Comment:
`List of local nameserver addresses`. Otherwise it gives an impression there's only a single nameserver (is there?)
It somewhat contradicts the comment on `EnvDNSLocalNameservers` if there's only one.